addon/content/options/options.html

```diff
@@ -38,6 +38,10 @@ <h3>Bugzilla</h3>
   <div class="service-settings" data-type="bugzilla">
     <label for="bugzilla-apiKey">API Key</label>
     <input type="password" id="bugzilla-apiKey" data-setting="apiKey"/>
+    <div class="form-rows">
+      <input type="checkbox" id="bugzilla-allBugzillaFlags" data-setting="allBugzillaFlags">
+      <label for="bugzilla-allBugzillaFlags">Count all Bugzilla Flags (implies "Count open needinfos too")</label>
+    </div>
```

Comment:
Presumably, the flags we care about are:

* `review`
* `feedback`
* `data-review`
* `needinfo`

Can we have those as individual entries instead?